js/apps/admin-ui/cypress/e2e/clients_test.spec.ts

```diff
@@ -231,6 +231,94 @@ describe("Clients test", () => {
     });
   });
 
+  describe("Client scopes evaluate subtab", () => {
+    const clientName = "testClient";
+
+    beforeEach(() => {
+      loginPage.logIn();
+      keycloakBefore();
+      commonPage.sidebar().goToClients();
+    });
+
+    before(() => {
+      adminClient.createClient({
+        protocol: "openid-connect",
+        clientId: clientName,
+        publicClient: false,
+      });
+    });
+
+    after(() => {
+      adminClient.deleteClient(clientName);
+    });
+
+    it("check effective protocol mappers list is not empty and find effective protocol mapper locale", () => {
+      commonPage.tableToolbarUtils().searchItem(clientName);
+      commonPage.tableUtils().clickRowItemLink(clientName);
+
+      clientDetailsPage.goToClientScopesTab();
+      clientDetailsPage.goToClientScopesEvaluateTab();
+
+      cy.findByTestId("effective-protocol-mappers")
+        .find("tr")
+        .should("have.length.gt", 0);
+
+      cy.findByTestId("effective-protocol-mappers").within(() => {
+        cy.contains("locale").should("exist");
+      });
+    });
+
+    it("check role scope mappings list list is not empty and find role scope mapping admin", () => {
+      commonPage.tableToolbarUtils().searchItem(clientName);
+      commonPage.tableUtils().clickRowItemLink(clientName);
+
+      clientDetailsPage.goToClientScopesTab();
+      clientDetailsPage.goToClientScopesEvaluateTab();
+      clientDetailsPage.goToClientScopesEvaluateEffectiveRoleScopeMappingsTab();
+
+      cy.findByTestId("effective-role-scope-mappings")
+        .find("tr")
+        .should("have.length.gt", 0);
+
+      cy.findByTestId("effective-role-scope-mappings").within(() => {
+        cy.contains("admin").should("exist");
+      });
+    });
+
+    it("check generated access token when user is not selected", () => {
+      commonPage.tableToolbarUtils().searchItem(clientName);
+      commonPage.tableUtils().clickRowItemLink(clientName);
+
+      clientDetailsPage.goToClientScopesTab();
+      clientDetailsPage.goToClientScopesEvaluateTab();
+      clientDetailsPage.goToClientScopesEvaluateGeneratedAccessTokenTab();
+
+      cy.get("div#generatedAccessToken").contains("No generated access token");
+    });
+
+    it("check generated id token when user is not selected", () => {
+      commonPage.tableToolbarUtils().searchItem(clientName);
+      commonPage.tableUtils().clickRowItemLink(clientName);
+
+      clientDetailsPage.goToClientScopesTab();
+      clientDetailsPage.goToClientScopesEvaluateTab();
+      clientDetailsPage.goToClientScopesEvaluateGeneratedIdTokenTab();
+
+      cy.get("div#generatedIdToken").contains("No generated id token");
```

Comment:
Same here, looks like a test id already exists.
```suggestion
      cy.findByTestId("generated-id-token-tab").contains("No generated id token");
```

Reply:
test id "generated-id-token-tab" is used for a right-side nav item, not for checking the <h2> text.

Reply:
Got it, ok. Then let's consider this resolved.